hi-ml-histopathology/testhisto/testhisto/mocks/tiles_generator.py

```diff
@@ -104,20 +104,20 @@ def generate_mock_histo_data(self) -> None:
         # to be able to create slide with different number of tiles.
         tiles, _ = next(iter(self.dataloader)) if self.dataloader else (None, None)
         for i, row in self.dataframe.iterrows():
-            slide_dir = self.tmp_path / f"{row[PandaTilesDataset.SLIDE_ID_COLUMN]}/train_images"
-            mask_dir = self.tmp_path / f"{row[PandaTilesDataset.SLIDE_ID_COLUMN]}/train_label_masks"
+            slide_dir = self.dest_data_path / f"{row[PandaTilesDataset.SLIDE_ID_COLUMN]}/train_images"
+            mask_dir = self.dest_data_path / f"{row[PandaTilesDataset.SLIDE_ID_COLUMN]}/train_label_masks"
             os.makedirs(slide_dir, exist_ok=True)
             os.makedirs(mask_dir, exist_ok=True)
 
             if self.mock_type == MockHistoDataType.PATHMNIST:
-                tile = tiles[i] * 255
+                tile = tiles[i]
             elif self.mock_type == MockHistoDataType.FAKE:
                 tile = torch.full(
                     fill_value=np.random.uniform(0, 255), size=(self.n_channels, self.tile_size, self.tile_size)
                 )
             else:
                 raise NotImplementedError
 
-            save_image(tile, str(self.tmp_path / row[PandaTilesDataset.IMAGE_COLUMN]))
+            save_image(tile.float(), str(self.dest_data_path / row[PandaTilesDataset.IMAGE_COLUMN]))
```

Comment:
Similarly here (and line 123), it is better to create variables for these paths for the purpose of a) clarifying what it represents and b) debugging